Formula/m/mapproxy.rb

```diff
@@ -17,25 +17,40 @@ class Mapproxy < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "26c4cd35b0765e40821998c324b0c31c438e90bf9e4defc915f771c1ad44d29f"
   end
 
-  depends_on "pillow"
+  # jpeg needed for pillow
+  depends_on "jpeg-turbo"
```

Comment:
There was some mention of keeping `pillow` in #157500 so may want to decide on that first.

Reply:
Thanks! Sounds good to me, I can also mark that note in a separate PR I have as well.